cime_config/tests.py

```diff
@@ -339,13 +339,13 @@
     "e3sm_bench_hires_f" : {
         "time"     : "01:00:00",
         "tests"    : (
-                 "PFS_P7200.ne120_ne120.F2010.eam-bench-noio",
-                 "PFS_P8640.ne120_ne120.F2010.eam-bench-noio",
-                 "PFS_P10800.ne120_ne120.F2010.eam-bench-noio",
-                 "PFS_P14400.ne120_ne120.F2010.eam-bench-noio",
-                 "PFS_P21600.ne120_ne120.F2010.eam-bench-noio",
-                 "PFS_P43200.ne120_ne120.F2010.eam-bench-noio",
-                 "PFS_P86400.ne120_ne120.F2010.eam-bench-noio",
+                 "PFS_P7200.ne120pg2_r05_oECv3.F2010.eam-bench-noio",
+                 "PFS_P8640.ne120pg2_r05_oECv3.F2010.eam-bench-noio",
+                 "PFS_P10800.ne120pg2_r05_oECv3.F2010.eam-bench-noio",
+                 "PFS_P14400.ne120pg2_r05_oECv3.F2010.eam-bench-noio",
+                 "PFS_P21600.ne120pg2_r05_oECv3.F2010.eam-bench-noio",
+                 "PFS_P43200.ne120pg2_r05_oECv3.F2010.eam-bench-noio",
+                 "PFS_P86400.ne120pg2_r05_oECv3.F2010.eam-bench-noio",
                  )
```

Comment:
In general we're trying to move to the EC30to60E2r2 for the low-res v2 grid. I'm guessing we don't have ne120pg2_r05_EC30to60E2r2 defined, but we can in order to update this test

Reply:
I'm getting an error with `SMS.ne120pg2_r05_EC30to60E2r2.F2010`:
```
ERROR: no alias ne120pg2_r05_EC30to60E2r2 defined
```
Please feel free to push updates to this branch or create a PR and we'll make this PR dependent on that PR.



Also, do we need to update these too?
1. high-res G-compset: `--res T62_oRRS18to6v3 --compset GMPAS-IAF`
2. low-res G-compset: `--res T62_oEC60to30v3 --compset GMPAS-IAF`
3. high-res WC-compset: `--res ne120pg2_r0125_oRRS18to6v3 --compset WCYCL1950`:
```
ERROR: grid alias ne120pg2_r0125_oRRS18to6v3 not valid for compset 1950SOI_EAM%CMIP6_ELM%SPBC_MPASSI_MPASO_MOSART_SGLC_SWAV_SIAC_SESP
```

Reply:
OK, we'll start a list -- and I'll check to see if the ocn/ice team wants to use oRRS18to6v3 for the high-res testing or the WC14to60E2r3. I'm pretty sure we want to move to EC30to60E2r2 for low-res. I'll push a fix for ne120pg2_r05_EC30to60E2r2 in just a minute

Reply:
For our official project metrics, the high-res F,G benchmarks need to be as consistent as possible so that we can measure progress across versions. So we should be trying to keep the RRS18to6 for those (and ne120 for atm).

Reply:
OK, let's leave oRRS18to6v3 for high-res and move to EC30to60E2r2 for low-res. Does that work for you, @philipwjones ?

Reply:
Sure @jonbob and if there's a more appropriate high-res mesh that will be used for production, we can modify our standard metrics for the next phase. Just needed at least one standard metric that we can measure progress against. Thx.

Reply:
@philipwjones - let's stick with the oRRS18to6v3 for now. The other is more of a regionally-refined mesh and I'm guessing that both will be used for different configurations.

Reply:
@amametjanov - the ne120pg2_r0125_oRRS18to6v3 alias had been set for:
````
compset="(DOCN|XOCN|SOCN|AQP1)"
````
I'll remove that limitation so it should work for WCYCL1950 and commit to this branch

Reply:
Instead of changing the grid, why not change the compset to F2010-CICE ?  Then you could keep ne120_ne120 and compare with older results.

Reply:
The grid ne120pg2_r0125_oRRS18to6v3 was originally for F-cases only. Need to remove the compset attribute for the grid.

Reply:
I'm getting an error while setting up `SMS.ne120pg2_ne120pg2.F2010-CICE`:
```
ERROR: Command: '/global/u2/a/azamat/cori/repos/E3SM-master/components/elm/bld/build-namelist -infile /global/cscratch1/sd/azamat/e3sm_scratch/cori-knl/tests/bench/20210917-sanity01/chk-05-f-cice/SMS.ne120pg2_ne120pg2.F2010-CICE.cori-knl_intel.20210923/Buildconf/elmconf/namelist  
-csmdata /global/cfs/cdirs/e3sm/inputdata 
-inputdata /global/cscratch1/sd/azamat/e3sm_scratch/cori-knl/tests/bench/20210917-sanity01/chk-05-f-cice/SMS.ne120pg2_ne120pg2.F2010-CICE.cori-knl_intel.20210923/Buildconf/elm.input_data_list 
-ignore_ic_year -namelist " &elm_inparm  start_ymd=00010101  /" 
-use_case 2010_CMIP6_control  -res ne120np4.pg2  -clm_start_type default 
-envxml_dir /global/cscratch1/sd/azamat/e3sm_scratch/cori-knl/tests/bench/20210917-sanity01/chk-05-f-cice/SMS.ne120pg2_ne120pg2.F2010-CICE.cori-knl_intel.20210923 
-l_ncpl 96 -lnd_frac /global/cfs/cdirs/e3sm/inputdata/share/domains/UNSET 
-glc_nec 0 -co2_ppmv 388.717 -co2_type diagnostic  -ncpl_base_period day  
-config /global/cscratch1/sd/azamat/e3sm_scratch/cori-knl/tests/bench/20210917-sanity01/chk-05-f-cice/SMS.ne120pg2_ne120pg2.F2010-CICE.cori-knl_intel.20210923/Buildconf/elmconf/config_cache.xml 
-bgc sp -mask gx1v6' 
failed with error 'File::Glob::glob() will disappear in perl 5.30. Use File::Glob::bsd_glob() instead. at /global/u2/a/azamat/cori/repos/E3SM-master/components/elm/bld/ELMBuildNamelist.pm line 3511.

ERROR : CLM build-namelist::ELMBuildNamelist::setup_cmdl_resolution() : res has a value ('ne120np4.pg2') that is NOT valid. 
Valid values are: '512x1024' '360x720cru' '128x256' '64x128' '48x96' '32x64' '8x16' '94x192' '0.23x0.31' '0.9x1.25' '1.9x2.5' '2.5x3.33' '4x5' '10x15' '5x5_amazon' '1x1_tropicAtl' '1x1_camdenNJ' '1x1_vancouverCAN' '1x1_mexicocityMEX' '1x1_asphaltjungleNJ' '1x1_brazil' '1x1_urbanc_alpha' '1x1_numaIA' '1x1_smallvilleIA' '0.1x0.1' '0.5x0.5' '3x3min' '5x5min' '10x10min' '0.33x0.33' 'ne4np4' 'ne4np4.pg2' 'ne11np4' 'ne16np4' 'ne30np4' 'ne30np4.pg2' 'ne60np4' 'ne120np4' 'ne240np4' 'ne1024np4.pg2' '1km-merge-10min' 'ne0np4_arm_x8v3_lowcon' 'ne0np4_conus_x4v1_lowcon' 'ne0np4_enax4v1' 'ne0np4_twpx4v1' 'r2' 'r05' 'r0125' 'NLDAS' 'ne0np4_northamericax4v1.pg2''
from dir '/global/cscratch1/sd/azamat/e3sm_scratch/cori-knl/tests/bench/20210917-sanity01/chk-05-f-cice/SMS.ne120pg2_ne120pg2.F2010-CICE.cori-knl_intel.20210923/Buildconf/elmconf'
```

Reply:
Don't do pg2.  Convert your existing tests to ne120_ne120.F2010-CICE.  Then also add ne120pg2_r05_oECv3.F2010 and run both until CICE really goes away.  Not sure how expensive that makes the whole thing.

Reply:
Getting an error setting up `SMS.ne120_ne120.F2010-CICE` too:
```
ERROR : CLM build-namelist::ELMBuildNamelist::add_default() : No default value found for fsurdat.
```